### PR TITLE
New tests for irm, test 

### PR DESCRIPTION
**Pull request by dezine2dev**
## Description

Unit tests for SDK fuse and irm

## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [x] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

### CODE REVIEW

**Comment by carlomazzaferro** [comment]
Make sure to merge with development so we can see the CI checks @dezine2dev

**Review by carlomazzaferro — CHANGES_REQUESTED** [review]
Run linting, otherwise this looks good to me

File: packages/sdk/tests/new/Fuse/irm/DAIInterestRateModelV2.spec.ts [hunk 1]
```diff
@@ -0,0 +1,177 @@
+import { BigNumber, constants, Contract, providers } from "ethers";
```
Comment (carlomazzaferro) [inline packages/sdk/tests/new/Fuse/irm/DAIInterestRateModelV2.spec.ts:1]: We don't really need this test, as we don't ever use the DAI IRM

Comment (carlomazzaferro) [reply packages/sdk/tests/new/Fuse/irm/DAIInterestRateModelV2.spec.ts:1]: We can keep it however

**Comment by dezine2dev** [comment]
> 

Updated lint issue.
Now I see hardhat issue which occurs in development as well

File: packages/sdk/tests-mocha/Fuse/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+describe("Fuse Index", () => {
+  let fuseBase: FuseBase;
+  let mockContract: SinonStubbedInstance<Contract>;
+  let mockFactory: SinonStubbedInstance<ContractFactory>;
+  beforeEach(() => {
+    mockContract = createStubInstance(Contract);
+    mockContract.connect.returns(mockContract);
+    Object.defineProperty(mockContract, "callStatic", {
+      value: {
+        getAllPools: stub().resolves({ length: 2 }),
```
Comment (carlomazzaferro) [inline packages/sdk/tests-mocha/Fuse/index.spec.ts:24]: Nice one!

**Comment by carlomazzaferro** [comment]
Awesome work @dezine2dev !

**Review by carlomazzaferro — APPROVED** [review]

**Pull request merged**